generators/unicodedata/cmd/src/fetcher.go

```diff
@@ -0,0 +1,81 @@
+package src
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// download the database files and save them locally
+
+const (
+	version            = "13.0.0"
+	urlUCDXML          = "https://unicode.org/Public/" + version + "/ucdxml/ucd.nounihan.grouped.zip"
+	urlUnicodeData     = "https://unicode.org/Public/" + version + "/ucd/UnicodeData.txt"
+	urlEmoji           = "https://unicode.org/Public/" + version + "/ucd/emoji/emoji-data.txt"
+	urlEmojiTest       = "https://unicode.org/Public/emoji/13.1/emoji-test.txt"
```

Comment:
This emoji test data seems different in kind to the other files, and is handled rather differently throughout the code. What is the difference there?

Reply:
Indeed it is kind of a special case, since it is not a data source, but rather a test sample, with a slightly different format. It is used to generate test data used by harfbuzz.